conductor_api/src/conductor/base.rs

```diff
@@ -472,6 +511,29 @@ impl Conductor {
         Dna::try_from(JsonString::from(contents))
     }
 
+    /// Default KeyLoader that actually reads files from the filesystem
+    fn load_key(file: &PathBuf) -> Result<Keypair, HolochainError> {
+        notify(format!("Reading agent key from {}", file.display()));
+
+        let mut file = File::open(file)?;
+        let mut contents = String::new();
+        file.read_to_string(&mut contents)?;
+
+        let passphrase = rpassword::read_password_from_tty(Some("Passphrase: "))?;
+
+        let bundle: KeyBundle = serde_json::from_str(&contents)?;
+        let mut passphrase = SecBuf::with_insecure_from_string(passphrase);
+        Keypair::from_bundle(
+            &bundle,
+            &mut passphrase,
+            Some(PwHashConfig(
+                pwhash::OPSLIMIT_INTERACTIVE,
+                pwhash::MEMLIMIT_INTERACTIVE,
+                pwhash::ALG_ARGON2ID13,
+            )),
```

Comment:
We should use `None` for the config here. We want to use the default SENSITIVE limits for the real solution.